### ISSUE TITLE
Invisible line numbers and first characters of the line.

### ISSUE THREAD

**Issue by lucasaf04** [microsoft/vscode#132913]
Does this issue occur when all extensions are disabled?: Yes

Version: 1.60.0
Commit: e7d7e9a9348e6a8cc8c03f877d39cb72e5dfb1ff
Date: 2021-09-01T10:54:53.442Z
Electron: 13.1.8
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Darwin x64 20.6.0 (Big Sur 11.5.2)

Steps to Reproduce:

1. Open this (for example):

```cpp
#include <iostream>
using std::cin;
using std::cout;
using std::endl;
#include <algorithm>
#include <list>

/**
 * @brief Main function.
 *
 * @return 0 upon exit success.
 */
int main()
{
    std::list<std::string> svec{};
    std::string str{};
    while (cin >> str)
    {
        svec.push_back(str);
    }

    int val{30};
    auto result{std::count(svec.cbegin(), svec.cend(), std::to_string(val))};
    cout << "The value " << val << " appears " << result << " times." << endl;

    return 0;
}
```

2. Check screenshot for details
<img width="2048" alt="Screenshot 2021-09-11 at 14 56 00" src="https://user-images.githubusercontent.com/39613949/132948680-ebc4ddfd-7772-48e6-8c37-fb03fc4e13a2.png">

**Comment by alexdima** [comment]
@lucasaf04 Does this reproduce when closing all vscode instances and launching `code --disable-gpu` ?

**Comment by lucasaf04** [comment]
Yes. Now even more characters are invisible.
<img width="2048" alt="Screenshot 2021-09-13 at 15 17 21" src="https://user-images.githubusercontent.com/39613949/133091182-e9a3631e-c592-4f7c-b8a6-6de86f020688.png">

**Comment by lucasaf04** [comment]
Now I have the same issue on my MacBook Pro M1 after updating to Big Sur 11.6 😭😭

**Comment by alexdima** [comment]
Have you seen something like this before @deepak1556  ?

**Comment by lucasaf04** [comment]
@alexdima 
When I updated to 11.6 I also updated my fonts (thing I had already	done on my iMac), and (what a coincidence) I'm using the buggy one
[Archive.zip](https://github.com/microsoft/vscode/files/7179254/Archive.zip)
The .ttf is the working one and the .ttf.bad is the buggy one.
If I use another font or the non-updated version VSCode works fine.

I spotted the problem booting my Mac in safe mode (that doesn't load user installed fonts).
Sorry for the inconvenience. Shall I close the issue?
